ui/react/pages/RuleProfile/validator.js

```diff
@@ -40,7 +40,14 @@ export const errorMessage = {
 export const validations = {
     priority: [
         {type: textValidations.isRequired, errorMessage: errorMessage.priorityRequired},
-        {type: textValidations.numberOnly, errorMessage: 'Priority should be a number'}
+        {type: textValidations.numberOnly, errorMessage: 'Priority should be a number'},
+        {type: textValidations.regex, value: /^[1-9][0-9]{0,9}/, errorMessage: 'Priority should be between 1 and 9999999999'},
+        {
```

Comment:
This new validation rule is duplicating what the `textValidations.regex` on line 44 is supposed to do (with `{0, 9}`), which probably is not working properly. As a result, this is not workarounding the problem, not fixing the original cause. I suggest to see why line 44 is not working. @behrindzhambaz

Reply:
![728](https://user-images.githubusercontent.com/44233695/141071913-be6025fa-d49c-4e50-a95c-e0f7347cd376.JPG)
textvalidations.length removed no need applied in regex validation